playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+
+Go ahead and click run!
+
+![](img/celebrate.gif)
+
+## Outline for the rest of this tutorial
+
+### Part I
+
+This first section is linear and more guided.
+
+In this section, you will learn to build an HTML5 app that, when you press a
+a button, it will call a phone.
+
+[Here is a demo of the app we will
+make](http://jsbin.com/fawuda/86/edit?html,css,js,console,output). To make this
+work for you:
+
+1. make sure you change change `555-555-5555` to your phone number
+2. click the "Run with JS" button in the output and enter your Twilio sid and
+   token
+
+Here is an outline of what you'll learn on the way:
+
+1. [Send text messages and make phone calls with one line of code]
+   (#send-text-messages-and-make-phone-calls)
+2. [Creating an account on JS Bin](#saving-our-work)
+3. [Call a phone when a button is clicked]
+   (#call-a-phone-when-a-button-is-clicked)
+
+If you're more advanced, feel free to skip this section and go to the next.
+Do make sure that you create a JS Bin account first though.
+
+### Part II
+
+This section is less guided. It showcases snippets of various APIs and how to
+use them.
+
+With you partner, look around to see what snippets you think are cool.
+Pick and choose some of them to build a mini project!
+
+We can break the tools into two categories:
+
+**Actions**
+
+Actions are just actions:
+
+- send a text message (addressed above)
+- call a phone and have a computer voice say something (addressed above)
+- call a phone and play music
+- play music through the web page
+
+**Triggers**
+
+Triggers run any code you want WHEN something happens. For example, you can
+run code:
+
+- When a button is pressed
+- When the phone is "waved" like a magic wand
+- When you receive a text message
+
+## Part I
+
+### Send text messages and make phone calls
+
+Go ahead and continue using the
+[JS Bin from the Quick Demo](http://jsbin.com/papawo/6/edit?js,console).
+
+If you've closed it already, open
+[the JS Bin](http://jsbin.com/papawo/6/edit?js,console)
+back up!
+
+If you havn't added your Twilio sid and token to this JS Bin, please do that
+by [following these directions](#adding-your-twilio-sid-and-token).
+
+We've already learned how to make a phone call and say something in the
+[Quick Demo](#quick-demo).
+
+Let's learn how to send text messages and call phones that play music.
+
+#### Sending Text Messages
+
+Type the below code as it is written _exactly_ in the JS Bin:
+
+```js
+Twilio.sendMessage("555-555-5555", "Evening upper east siders...");
+```
+
+First change `555-555-5555` to your own phone number. Feel free to change the
+message too.
+
+> ![](img/text_1.gif)
+
+Go ahead and click "Run" again.
+
+> ![](img/text_2.gif)
+
+Because I left the code that calls my phone, I get a phone call as well
+as a text message.
+
+> If it doesn't, ask a neighboring group to see if they can see what's wrong.
+> Otherwise, raise your hand to ask a facilitator for help!
+
+![](img/celebrate.gif)
+
+### Saving Our Work
```

Comment:
Maybe move this section into a separate file in the root `playbook` folder. It is pretty key to a lot of projects, and thus should be able to be linked too.